open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/buffergraph/BufferGraphConnector.java

```diff
@@ -72,124 +60,129 @@ public void setMainGraph(Object mainGraph) {
     }
 
     @Override
-    public void addEntity(LineageEvent lineageEvent){
+    public void addEntity(LineageEvent lineageEvent) {
 
-        GraphTraversalSource g =  bufferGraph.traversal();
+        GraphTraversalSource g = bufferGraph.traversal();
 
         Set<GraphContext> verticesToBeAdded = new HashSet<>();
         lineageEvent.getAssetContext().entrySet().stream().forEach(entry ->
                 {
-                    if(entry.getValue().size()>1){
+                    if (entry.getValue().size() > 1) {
                         verticesToBeAdded.addAll(entry.getValue());
-                    }else {
+                    } else {
                         verticesToBeAdded.add(entry.getValue().stream().findFirst().get());
                     }
                 }
         );
 
-        verticesToBeAdded.stream().forEach(entry -> addVerticesAndRelationship(g,entry));
+        verticesToBeAdded.stream().forEach(entry -> {
+            try {
+                addVerticesAndRelationship(g, entry);
+            } catch (JanusConnectorException e) {
+                log.error("An exception occured", e);
```

Comment:
The message of the log might be more relevant for the person see it.
Ex: 
log.error("An exception occurred while adding Vertices and relationship ", e);